chain/jsonrpc/src/lib.rs

```diff
@@ -958,6 +968,23 @@ impl JsonRpcHandler {
     }
 }
 
+#[cfg(feature = "sandbox")]
+impl JsonRpcHandler {
+    async fn sandbox_patch_state(
+        &self,
+        patch_state_request: near_jsonrpc_primitives::types::sandbox::RpcSandboxPatchStateRequest,
+    ) -> Result<Value, RpcError> {
+        actix::spawn(
+            self.client_addr
+                .send(NetworkClientMessages::Sandbox(NetworkSandboxMessage::SandboxPatchState(
+                    patch_state_request.records,
+                )))
+                .map(|_| ()),
+        );
```

Comment:
Why do we want to spawn the task in background instead of awaiting till it completes before responding the client with any status?

Reply:
Yeah awaiting the change would be easier for use, I'm figuring out how to do it

Reply:
Try this:

```suggestion
        self.client_addr
            .send(NetworkClientMessages::Sandbox(NetworkSandboxMessage::SandboxPatchState(
                patch_state_request.records,
            ))).await?;
```